src/uwtools/resources/jsonschema/shave.jsonschema

```diff
@@ -0,0 +1,39 @@
+{
+  "properties": {
+    "shave": {
+      "additionalProperties": false,
+      "properties": {
+        "config": {
+          "additionalProperties": false,
+          "properties": {
+            "input_grid_file": {
+              "type": "string"
+            },
+            "nh4": {
+              "type": "integer"
+            },
+            "nx": {
+              "type": "integer"
+            },
+            "ny": {
+              "type": "integer"
+            }
+          }
+        },
```

Comment:
Are all of these optional?

Reply:
Ah, I went through a few versions and looks like I didn't re-require them in the end, thanks! I'll add that to the test too